### ISSUE TITLE
Virtual Pages: Update subtitle and tooltip copies for homepage

### ISSUE THREAD

**Issue by fushar** [Automattic/wp-calypso#70466]
## Description

We want to emphasize the fact that the virtual homepage page is served by a template and is editable in the Site Editor.

The template name will be shown in the subtitle, which could be either `Front Page`, `Home`, or `Index`.

@Automattic/ganon @SaxonF 

## Screenshots

| Before | After |
| ---- | ---- |
| <img width="492" alt="image" src="https://user-images.githubusercontent.com/1525580/204215753-8232bfda-8a07-4245-888e-c7d7ac5309c0.png"> | ![image](https://user-images.githubusercontent.com/1525580/204215235-33e7b26d-91b4-4c92-b7bb-f82e5f188418.png) |
| <img width="478" alt="image" src="https://user-images.githubusercontent.com/1525580/204215688-85b94219-1d11-4a56-aab3-555d4277d9a3.png"> |  <img width="402" alt="image" src="https://user-images.githubusercontent.com/1525580/204215571-dd3360d9-6f7d-4425-876a-775193f5be3e.png"> |

**Comment by autumnfjeld** [comment]
@fushar @SaxonF  I assume the copy will change with the template. e.g.   "Showing the Home template".  "Showing the Index template"  Is that correct? If so could you update the issue description to make this clear?  Thank you.

**Comment by fushar** [comment]
@autumnfjeld Yes it's correct, thanks! I've updated the description.

**Comment by autumnfjeld** [comment]
@fushar. "Showing the Front Page template", "showing" sounds weird to me.  And the flow, as a user would read it: "Homepage, Showing the Front Page template." doesn't make sense.  

We can use better phrasing.  Sorry I didn't catch this earlier.

Learn more goes out to https://wordpress.com/support/templates/#template-hierarchy, correct? I looked there for guidance on a better phrase.  I also looked at other usage of page & template together
![image](https://user-images.githubusercontent.com/5665959/205775366-3054cd27-1a22-41a0-b449-7956ace125e3.png)



Let's use:

" Your homepage uses the Front Page template."

It would be fine to make a new issue for this copy update if that is easier.  If you want to merge this and then later change the copy so we don't block progress that is fine.
